src/main/java/gmailgdogra/Officer.java

```diff
@@ -0,0 +1,73 @@
+package gmailgdogra;
+
+public class Officer {
+
+    private String firstName;
+    private String lastName;
+    private Location location;
+
+    public Officer(String firstName, String lastName, Location location) {
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.location = location;
+    }
+
+    public Officer(String firstName, String lastName) {
```

Comment:
using this public constructor you leave the location null. Consider UNKNOWN location instead